### DISCUSSION TITLE
Please bring back "X starred your repo" in some shape or form

### DISCUSSION THREAD

**Discussion by Tyrrrz** [community/community#62580] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The previous feed notified you when someone starred or forked your repo. It was a good way to see if one of your projects had sudden spikes of attention (e.g., someone made a post on Hackernews) and just in general a nice way to gauge their popularity.

Now, I understand why you decided to revamp the feed. However, none of the presented information is valuable to me in any way. I don't really care about whether a repository I starred 5 years ago had a new release today. The "trending repositories" rarely contain anything I'd actually be interested in. And the worst part is that the activity on your own repositories (such as when someone gives a star) is no longer visible.

My suggestion is to please bring back personal repo activity as a new category in this new feed screen. Thank you.

**Comment by diyaryvzz** [comment]
Thanks
